src/components/common/Certify.vue

```diff
@@ -3,20 +3,75 @@
     <header>
       <h2>2. Certify</h2>
     </header>
-
     <span>Enter the legal name of the person authorized to complete and submit these changes.</span>
-
-    <div id="certify-container" class="mt-4">
-      Under construction!
-    </div>
+    <certify
+      :currentDate="readableCurrentDate"
+      :certified-by="getCertifyState.certifiedBy"
+      :is-certified="getCertifyState.valid"
+      :entity-display="readableEntityType"
+      :message="certifyMessage"
+      @update:certifiedBy="onCertifiedBy($event)"
+      @update:isCertified="onValid($event)"
+    />
   </section>
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator'
+import { Component, Mixins } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+
+// Components
+import { Certify } from '@bcrs-shared-components/certify'
+
+// Interfaces & Mixins
+import { DateMixin, EnumMixin } from '@/mixins'
+import { ActionBindingIF, CertifyIF } from '@/interfaces'
+import { EntityTypes } from '@/enums'
+
+@Component({
+  components: {
+    Certify
+  }
+})
+export default class CertifySection extends Mixins(DateMixin, EnumMixin) {
+  @Getter getCertifyState!: CertifyIF
+  @Getter getCurrentDate!: string
+  @Getter getEntityType!: EntityTypes
+
+  @Action setCertifyState!: ActionBindingIF
+
+  readonly certifyMessage = 'Note: It is an offence to make a false of misleading statement in respeoct of a material' +
```

Comment:
Do we still have "resources" somewhere that contain these strings for different entity types?

Reply:
o geez, yea we do i think. Didn't even cross my mind. 
Will cross reference to see if that matches the current design and implement if so!

Reply:
Added in latest commit!

Reply:
Awesome!